### PR TITLE
Process Routebook Loops

### PR DESCRIPTION
**Pull request by kevinchen220**

### CODE REVIEW

File: benefitCostScore/convertData.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+from email import header
+import pandas as pd
+
+
+def convertData(file):
```
Comment (aliciabremer) [inline benefitCostScore/convertData.py:5]: Code and tests look great - good work! To possibly be able to reuse code in the future and for clarity, my one piece of feedback would be to split the tasks in convertData into smaller functions that get called - for example, into functions that clean up speed data, get number of stops, etc. That way, these functions can also be tested individually given smaller pieces of data!

**Review by aliciabremer — CHANGES_REQUESTED** [review]

**Review by probro27 — CHANGES_REQUESTED** [review]
Hey @kevinchen220 good job separating the functions. Could you add type annotations to the parameters so that code is better documented for people to use.
Also if possible could you add tests for all of the smaller functions. 
Thanks.

**Review by probro27 — APPROVED** [review]
LGTM! 
I'll let Alicia review the code first before merging but I think it's pretty good. 
The poetry.lock issue will be resolved soon. Then you can merge in the PR.

**Review by probro27 — APPROVED** [review]
LGTM! 
I'll let Alicia review the code first before merging but I think it's pretty good. 
The poetry.lock issue will be resolved soon. Then you can merge in the PR.

**Comment by aliciabremer** [comment]
This looks good to me too! Nice work Kevin!

**Review by aliciabremer — APPROVED** [review]

**Comment by probro27** [comment]
@kevinchen220 Sorry to get back to you on this. So I automated the tests that you wrote so that they run whenever a new pull request comes in. Looks like your test fail, so you might wanna check it out, before we can approve the pull request. I resolved all of the merge conflicts so first please do `git pull` on this branch and then start working on figuring out the errors in the tests.

**Pull request merged**